apps.toml

```
@@ -3980,6 +3980,11 @@ level = 8
 state = "working"
 url = "https://github.com/YunoHost-Apps/readeck_ynh"
 
+[readflow]
+category = "small_utilities"
+state = "working"
+url = "https://github.com/YunoHost-Apps/readflow_ynh/"
```

Maybe the trailing slash?

```suggestion
url = "https://github.com/YunoHost-Apps/readflow_ynh"
```

---

God catch! Thanks 🎉